app/views/adoption_application/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Pet Adoption Application:</h2>
+<section class="form">
+  <%= form_tag("/favorites", method: "post") do %>
+    <%= label_tag("Pets To Apply For: ") %><br>
+    <% @favorites.each do |favorite_id| %>
+      <% favorite = Pet.find(favorite_id) %>
+      <%= check_box_tag(:pets, value = favorite_id) %>
+      <%= label_tag("#{favorite.name}")%>
+    <% end %><br />
```

Comment:
Spend over 2 hours on just this little part, because the favorite_id needs to get assigned to the :pets param in order to get passed to the controller to create the PetApplication. Currently, though I can display check boxes for all the favorite pets, and a user can check multiple boxes, I can only pass one :id to the controller as :pets. Hopefully we can come back to this and build it out to complete the functionality, but for now I've listed it as an issue to come back to.

Reply:
That was a long process! For sure let's revisit this if we have the time.